app/views/api-docs/examples/index.html.md

```diff
@@ -0,0 +1,191 @@
+# e-Manifest API Examples | US EPA
+
+Code examples of using the e-Manifest API.
+
+Using the e-Manifest API consists of two major steps: uploading and signing manifests.
+
+The production base URL is **https://e-manifest.18f.gov/api/v0**
+
+Notation note: path variables are expressed as `:variable_name`. The colon is not part of the URI.
+
+## <a name="submit-manifest"></a>Submit a manifest
+
+```bash
+curl -i -X POST -H 'Content-Type: application/json' \
+  --data @manifest.json \
+  https://e-manifest.18f.gov/api/v0/manifests?tracking_number=:manifest_tracking_number
+```
+
+where the `:manifest_tracking_number` is from Line 4 of form 8700-22.
+
+An example `manifest.json` file looks like:
+
+```json
+<%= render 'examples/manifest.json' %>
+```
+
+The response from that POST will include no body content, but the `Location` header will contain
+the URL of the submitted manifest, which will include the unique e-Manifest identifier. A successful
+submission should return a 201 status.
+
+## <a name="fetch-manifest"></a>Fetch a manifest
+
+To retrieve a previously submitted e-Manifest, you need the e-Manifest ID from
+the [Submit a manifest example](#submit-manifest).
+
+```bash
+curl -i -X GET https://e-manifest.18f.gov/api/v0/manifest?id=:manifest_id
```

Comment:
I think this should be `https://e-manifest.18f.gov/api/v0/manifest/:uuid` bc of route `get 'manifests/:id', to: 'manifests#show'`